src/Mvc/Mvc.ViewFeatures/src/Rendering/ViewContext.cs

```diff
@@ -15,20 +17,22 @@ namespace Microsoft.AspNetCore.Mvc.Rendering
     /// </summary>
     public class ViewContext : ActionContext
     {
-        private FormContext _formContext;
-        private DynamicViewData _viewBag;
-        private Dictionary<object, object> _items;
+        private FormContext _formContext = default!;
+        private DynamicViewData? _viewBag;
+        private Dictionary<object, object?> _items = default!;
 
         /// <summary>
         /// Creates an empty <see cref="ViewContext"/>.
         /// </summary>
         /// <remarks>
         /// The default constructor is provided for unit test purposes only.
         /// </remarks>
+#nullable disable warnings
```

Comment:
Work in progress?

Reply:
The constructor is meant for unit tests where none of the properties are initialized. It was either skipping nullability for this constructor or lots of `default!`. Should I just do the former?

Reply:
Ok! Have a comment explaining why nullable is disabled

Reply:
```suggestion
#nullable disable warnings
        // This is a unit-test only constructor where no property is initialized. We'll avoid having to
        // use null-forgiveness operator by simply skipping nullable warnings on this constructor.
```